homeassistant/components/logbook.py

```diff
@@ -286,6 +289,26 @@ def humanify(hass, events):
                     'context_user_id': event.context.user_id
                 }
 
+            elif event.event_type == EVENT_HOMEKIT_CHANGED:
+                data = event.data
+                entity_id = data[ATTR_ENTITY_ID]
+
+                value_msg = " to {}".format(
+                    data.get[ATTR_VALUE]) if data.get[ATTR_VALUE] else ''
```

Comment:
I think that I might have mislead you here. Sorry for that.
For value it is probably better to define it beforehand:
```py
value = data.get('ATTR_VALUE`)

value_msg = " to {}".format(value) if value else ''
```

For `ATTR_SERVICE` and `ATTR_DISPLAY_NAME` below the formatting is correct.

Another point probably also unnecessary, since we'll have use `get` for `value` can you use it for `entity_id` as well? Just looks a bit better :)

The empty line between `value_msg` and `message` would be unnecessary.

Reply:
Done :-)